straxen/plugins/nveto_ext_timings.py

```diff
@@ -0,0 +1,69 @@
+import numpy as np
+from copy import copy
+from immutabledict import immutabledict
+
+import strax, straxen
+
+export, __all__ = strax.exporter()
+
+@export
+@strax.takes_config(
+    strax.Option('channel_map', track=False, type=immutabledict,
+                 help="immutabledict mapping subdetector to (min, max) "
+                      "channel number."),
+)
+class nVetoExtTimings(strax.Plugin):
+    """
+    Plugin which computes the time differnce from external trigger timing to hitlets_nv.
+    The nearlest `time` of `raw_records_nv` before `hitlets_nv` time are used as the
+    external trigger timings.
+
+    Note:
+        Channel dependence for trigger timing has been ignored. It should be the same,
+        But unfortunetely not.
+    """
+    depends_on = ('raw_records_nv', 'hitlets_nv')
+    provides = 'ext_timings_nv'
+    data_kind = 'hitlets_nv'
+
+    compressor = 'zstd'
+    ends_with = '_nv'
+    __version__ = '0.0.1'
+
+    def infer_dtype(self):
+        dtype = copy(strax.time_dt_fields)
```

Comment:
```suggestion
        dtype = []
        dtype += strax.time_dt_fields
```
Like this you do not need the additional import.